### ISSUE TITLE
Catch error when clingo is not in $PATH

### ISSUE THREAD

**Issue by cfrioux** [Aluriak/predator#1]
So far the error is the following: 
```
FileNotFoundError: [Errno 2] No such file or directory: 'clingo': 'clingo'
```
Ideally the program should exit smoothly rather than crash.

**Comment by Aluriak** [comment]
I'm on it !

Can i have the full stacktrace ?

**Comment by cfrioux** [comment]
Sure! 
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "..../predator/predator/predator/__main__.py", line 96, in <module>
    for idx, seeds in enumerate(predator.search_seeds(graph, seeds, forbidden_seeds, targets), start=1):
  File "..../predator/predator/predator/predator.py", line 42, in search_seeds
    yield from func(graph_data, start_seeds, forbidden_seeds, targets, graph_filename)
  File "..../predator/predator/predator/predator.py", line 78, in search_seeds_activate_all
sccs, scc_dag = compute_sccs(graph_data, graph_filename=graph_filename)
  File "..../predator/predator/predator/predator.py", line 110, in compute_sccs
    models = clyngor.solve(ASP_SRC_ENUM_CC, inline=graph_data)
  File "/home/venv/lib/python3.6/site-packages/clyngor/solving.py", line 106, in solve
    shell=bool(subproc_shell),
  File "/usr/lib/python3.6/subprocess.py", line 709, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1344, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'clingo': 'clingo'
```

**Comment by Aluriak** [comment]
Fixed with c9badbff: the implementation the smooth exit is in [`utils.solve`](https://github.com/Aluriak/predator/blob/master/predator/utils.py#L18). Change as needed.
